common/test/acceptance/pages/lms/create_mode.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, browser, course_id, mode_slug=None, mode_display_name=None, m
         """The mode creation page is an endpoint for HTTP GET requests.
 
         By default, it will create an 'honor' mode for the given course with display name
-        'Honor Code', a minimum price of 0, no suggested prices, and using USD as the currency.
+        'Honor Code', a minimum price of 0, no suggested prices, and using configured currency as the currency.
```

Comment:
```suggestion
        'Honor Code', a minimum price of 0, no suggested prices, and using the configured currency as the currency.
```

Reply:
Reverting this file changes as this file is in the test directory so we should not change it.